tests/functional/UseCase/Table/CreateDropTableTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\FunctionalTests\UseCase\Table;
+
+use Keboola\StorageDriver\Command\Project\CreateProjectResponse;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableResponse;
+use Keboola\StorageDriver\Command\Table\DropTableCommand;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\FunctionalTests\BaseCase;
+use Keboola\StorageDriver\Teradata\Handler\Table\Create\CreateTableHandler;
+use Keboola\StorageDriver\Teradata\Handler\Table\Drop\DropTableHandler;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+class CreateDropTableTest extends BaseCase
+{
+    protected GenericBackendCredentials $projectCredentials;
+
+    protected CreateProjectResponse $projectResponse;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        [$credentials, $response] = $this->createTestProject();
+        $this->projectCredentials = $credentials;
+        $this->projectResponse = $response;
+    }
+
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        $this->cleanTestProject();
+    }
+
+    public function testCreateTable(): void
+    {
+        $tableName = 'TableName';
+
+        $handler = new CreateTableHandler();
+        $command = (new CreateTableCommand())
+            //->setPath([])
+            ->setTableName($tableName)
+            ->setColumns([
+                // TODO path
+                // TODO nullable
+                (new CreateTableCommand\TableColumn())
+                    ->setName('id')
+                    ->setType('integer')
+                    ->setNullable('false'),
+                (new CreateTableCommand\TableColumn())
+                    ->setName('name')
+                    ->setType('varchar')
+                    ->setLength('50')
+                    ->setNullable('false'),
+            ])
+            ->setPrimaryKeysNames(['id']);
+
+        $response = $handler(
+            $this->projectCredentials,
+            $command,
+            []
+        );
+        $this->assertInstanceOf(CreateTableResponse::class, $response);
+
+        $db = $this->getConnection($this->projectCredentials);
+
+        // test table structure
+        $tableInfo = $db->fetchOne(sprintf(
+            'SHOW TABLE %s;',
+            $tableName
+        ));
+        $this->assertStringContainsString('CREATE SET TABLE MJ_CONNECTION_DEV.Employees', $tableInfo);
+        $this->assertStringContainsString('id INTEGER,', $tableInfo);
+        $this->assertStringContainsString('name VARCHAR(30) CHARACTER SET LATIN NOT CASESPECIFIC,', $tableInfo);
+        $this->assertStringContainsString('UNIQUE PRIMARY INDEX ( id )', $tableInfo);
```

Comment:
Tady sa v tom zbytečně plácáš a nebude to fungovat.
Použij `TeradataTableReflection` a zkontroluj jestli je všecko v cajku podle ní.

Reply:
jo, to byla prvni verze, dal jsem pryc

Reply:
To bylo dobře že sa to testovalo enom bys to neměl "bastlit" takto na koleni, ale použít tu `TeradataTableReflection`.
Ber to tak, žes odpálil nějaký cmd a musíš ověřit, že udělal to co má. Takže bys měl ověřit, že má ta tabulka všecky sloupce správně.